packages/sado-connect/src/lib/signPsbt.ts

```diff
@@ -55,7 +58,16 @@ export default async function signPsbt({
     const xverseSignPsbtOptions = {
       psbt,
       network,
-      inputs: [],
+      inputs: [
+        {
+          address,
+          signingIndexes:
+            options?.signingIndexes ??
+            psbt.data.inputs.map((value, index) => index), // If signingIndexes is not provided, just sign everything
```

Comment:
can we extract this out to a const above maybe like `const allInputsIndex = psbt.data.inputs.map((value, index) => index)`

Reply:
Ideally, the map should not run if `signingIndex` is provided.

Will refactor to below:
```
const getAllInputIndices = () => psbt.data.inputs.map((value, index) => index);

inputs: [ { address, signingIndexes: options?.signingIndexes ?? getAllInputIndices() } ]
```